Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -158,6 +160,8 @@ class BrowserViewController: UIViewController,
         return NewTabAccessors.getNewTabPage(profile.prefs)
     }
 
+    lazy var isReduxIntegrationEnabled: Bool = ReduxFlagManager.isReduxEnabled
```

Comment:
I'm not sure what's the benefit of integrating a feature flag that's already enabled on all channels? IMO we should remove this flag soon since it's already used in RemoteTabsPanel and ThemeSettingsController. If we want to have the benefit of having the possibility to turn redux off for FakeSpot, it should be its own flag - otherwise turning this flag off has multiple side effects.

Reply:
The reason we are being extra cautious with this is because of the increase of hangs in v119 where Redux was released so until is excluded as a possible cause we decided to use the flag but we hope we can remove it before releasing this feature

Reply:
Ok thank you 🙌